components/sidebar/sidebar_channel/sidebar_direct_channel/index.ts

```diff
@@ -53,6 +54,7 @@ function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
         redirectChannel,
         active,
         botIconUrl: botIconImageUrl(teammate),
+        isMobile: isMobile(),
```

Comment:
Nit: Don't need to pass this through the redux connector, we can just check this directly in the component.

Reply:
If it goes through redux it updates as you resize the window though, otherwise you need a reload 🤷‍♀️